ambiata-mafia.cabal

```diff
@@ -22,6 +22,7 @@ library
                     , attoparsec                      >= 0.12       && < 0.14
                     , base16-bytestring               == 0.1.*
                     , bytestring                      == 0.10.*
+                    , Cabal                           >= 1.22       && < 1.25
```

Comment:
I think it shouldn't be too much of an issue in this case, because we're using a fairly stable part of the API here, but I'd like to keep it to a minimum.